cross/build-rootfs.sh

```diff
@@ -60,12 +62,6 @@ for i in "$@" ; do
             __UbuntuArch=i386
             __UbuntuRepo="http://archive.ubuntu.com/ubuntu/"
             ;;
-        lldb3.6)
```

Comment:
We want to keep the cross-rootfs generation scripts consistent across repos as we use them to generate docker images - please undo this change.